layouts/products/energy-id.html

```
@@ -0,0 +1,96 @@
+      <a class="project-item-small wrapped"><i class="portrait huge icon project-item-white"></i></a>
+      <div class="title-one">{{.Params.Title}}</div>
+    </div>
+    <div class="data-content-one">
+
+      <div id="energy-id-preamble">
+        {{ .Content }}
+      </div>
+
+      <div class="clear"></div>
+
+      <h2>Software Database</h3>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </h3> ]

(tag-pair)